doc/whats_new/v1.4.rst

```diff
@@ -187,6 +187,13 @@ Changelog
   ``**score_params`` which are passed to the underlying scorer.
   :pr:`26525` by :user:`Omar Salman <OmarManzoor>`.
 
+:mod:`sklearn.metrics`
+......................
+
+- |Enhancement| :func:`metrics.f_regression` and :func:`metrics.r_regression` now
+  supports SciPy sparse arrays.
+  :pr:`27239` by :user:`Tialo <Tialo>`.
```

Comment:
Feel free to update your name (only the first mention since the second one link to your GitHub handle). This is also fine keeping it as-is.